app/src/main/res/values/strings.xml

```diff
@@ -45,6 +45,9 @@
     <string name="select_ca_failed">Failed to read CA: %s</string>
     <string name="login">Login</string>
     <string name="check_url">Check URL</string>
+    <string name="permissions_dialog_grant">Grant</string>
+    <string name="permissions_denied_temp">Gotify requires permission to display push notifications.</string>
```

Comment:
I've made this a little more specific.

Reply:
To be exact it requires the exact alarm permission as well but you don't see a permission dialog atm as it given right away.
But yeah, possible for now.